routes.js

```diff
@@ -1,86 +1,91 @@
-// const express = require('express');
-// const createMyCustomApi = express.Router();
-// const fs = require('fs');
+const express = require('express');
+const membersAPI = express.Router();
+const fs = require('fs');
 
+let data = require('./members_Array.json');
 
+const saveData = (data) => {
+  fs.writeFileSync('members_Array.json', JSON.stringify(data, null, 2));
+};
 
 
-// // Define routes
-// createMyCustomApi.get('/getAllMembers', (req, res) => {
-//   fs.readFile("./members_Array.json",  (err, data) => {
-//     if(err){
-//       res.status(500).send(err.message)
-//     }
-//     res.status(200).send(data.toString());
-//   });
-// });
+// New function to load data
+const loadData = () => {
+  const rawData = fs.readFileSync('./members_Array.json');
+  return JSON.parse(rawData);
+};
```

Comment:
Great! To make current this file focused on `routes`. Could you please make a next step and extract that logic into a file named like `dataAccess.js` or `membersFileAdapter` or `membersDataAccess`?

Reply:
Done